test/flows/check_benefits_support_flow_test.rb

```diff
@@ -455,5 +455,12 @@ class CheckBenefitsSupportFlowTest < ActiveSupport::TestCase
         assert_rendered_outcome text: "Check if you’re eligible for 30 hours free childcare"
       end
     end
+
+    should "render Carer's Allowance when eligible" do
+      add_responses carer_disability_or_health_condition: "yes"
+
+      assert_rendered_outcome text: "Carer's Allowance"
```

Comment:
This is unrelated to the changes you are making, but the apostrophe in Carer's Allowance should be a typographic quote, i.e. `’`.  I think this one was missed by accident as the other quotes seem to be correct.

Reply:
good spot!